newrelic.js

```diff
@@ -14,6 +14,21 @@ exports.config = {
    * Your New Relic license key.
    */
   license_key: 'license key here',
+  /**
+   * Controls the method of cross agent tracing in the agent.
```

Comment:
I see this took the language from the `default.js` file and moved it here. I think we probably should improve that language (and tweak the default as well but that can be done separate from this). We generally don't use the terms "cross agent tracing" AKA CAT when referring to Distributed Tracing. Seems like this could be confusing, particularly since there's other settings for CAT. I'd probably omit this line.